src/index.html

```diff
@@ -1,9 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
- Kindly enable your linters into your project. 
- Make sure that there are no linter errors.

Reply:
Hello, I've already done that right now, but I'm getting an error from Lighthouse. Can you take a look again and give me advice on how I can solve this?
Thank you for your time.

Reply:
Hi @Zibilyonik, how are you? check your linter files if you set up that files correctly.